src/CSF.SRDashboard.Client/Components/HelpSectionComponent.razor.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.AspNetCore.Components;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CSF.SRDashboard.Client.Components
+{
+    public partial class HelpSectionComponent
+    {
+        [Parameter]
+        public string SectionHeader { get; set; }
+        [Parameter]
+        public string SectionId { get; set; }
+        [Parameter]
+        public RenderFragment HelpAnswer { get; set; }
+
+        public string buttonclass
+        {
+            get
+            {
+                switch (this.isCollapsed)
+                {
+                    case false: return "fas fa-minus";
+                    default: return "fas fa-plus";
+                }
+            }
+            set { }
+        }
```

Comment:
We can replace the whole thing with this:
```suggestion
        public string buttonClass => this.isCollapsed ? "fas fa-plus" : "fas fa-minus";
```